corehq/apps/api/odata/tests/data/populated_case_metadata_document.xml

```diff
@@ -1,22 +1,6 @@
  <edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.0">
    <edmx:DataServices>
      <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="CommCare">
-       <ComplexType Name="CaseProperties:case_type_with_no_case_properties">
-         <Property Name="backend_id" Type="Edm.String"/>
-         <Property Name="case_name" Type="Edm.String"/>
-         <Property Name="case_type" Type="Edm.String"/>
-         <Property Name="date_opened" Type="Edm.String"/>
-         <Property Name="owner_id" Type="Edm.String"/>
-       </ComplexType>
-       <ComplexType Name="CaseProperties:case_type_with_case_properties">
-         <Property Name="backend_id" Type="Edm.String"/>
-         <Property Name="case_name" Type="Edm.String"/>
-         <Property Name="case_type" Type="Edm.String"/>
-         <Property Name="date_opened" Type="Edm.String"/>
-         <Property Name="owner_id" Type="Edm.String"/>
-         <Property Name="property_1" Type="Edm.String"/>
-         <Property Name="property_2" Type="Edm.String"/>
```

Comment:
I'm not sure it matters, and I think I'm just misunderstanding something, but why were there underscores in these to begin with? Should they not have previously been replaced with empty strings?

Reply:
The test mocked out the function that remove the empty strings, which is why they got passed through to the metadata document.